CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_iterable v0.13.0
+[PR #51](https://github.com/fivetran/dbt_iterable/pull/51) includes the following updates:
+
+## Breaking Changes (`--full-refresh` required after upgrading)  
+- Added a field called `first_open_or_click_event_at` in the `iterable__user_campaign` model. This timestamp shows the first time a user interacted with a campaign, recording the earliest occurring event out of 'emailOpen', 'emailClick', and 'pushOpen'. ([PR #50](https://github.com/fivetran/dbt_iterable/pull/50))
+- Corrected the incremental filter in `iterable__events` model to now use the `created_on` date field instead of the `created_at` timestamp. 
+   - Previously, this would potentially exclude late-arriving new records from populating in the end models if they had an older `created_at` value than what was present in the model. Switching to `created_on` widens the criteria.
+- Updated upstream `stg_iterable__user_history` model from materializing as a table to a view in order to improve performance.
+- In order to ensure no issues a `--full-refresh` is required after upgrading.
+
+## Under the Hood
+- In addition to using `created_on` in the incremental logic in `iterable__events`, we introduced a `iterable_lookback_window` variable to increase the window for accommodating potential late-arriving records. The default is 7 days prior to the maximum `created_on` value present in the `iterable__events` model, but you may customize this by setting the var `iterable_lookback_window ` in your dbt_project.yml. See the [Lookback Window section of the README](https://github.com/fivetran/dbt_iterable/blob/main/README.md#lookback-window) for more details.
+- Added a section in the [README]((https://github.com/fivetran/dbt_iterable/blob/main/README.md#pivoting-out-event-metrics)) about the `iterable__event_metrics` variable and how to use it to specify which event metrics to pivot out.
+- Removes `created_on` from the uniqueness test in `iterable__events`. Uniqueness is now tested solely on `unique_event_id`, a surrogate key made up of `event_id` (`_fivetran_id` in the raw table, which is a Fivetran-created unique identifier derived from hashing campaign_id, created_at, and event_name) and `_fivetran_user_id` (a Fivetran-created column derived from a hash of `user_id` and/or `email`).
+- Modified the `event` seed data to more accurately represent real-life data, with a unique `_fivetran_id` for each campaign_id, created_at, and event_name.
+
```

Comment:
@fivetran-reneeli Looks like you made the same documentation update in `iterable.yml` that you did in the source yml files, should this be added?

```suggestion
## Documentation Update
- Updates the descriptions of timestamp-based fields. Previously they were described as milliseconds since epoch time, but they should be standard timestamps.

```